docs/serverless/05-01-serverless.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Serverless chart
+type: Configuration
+---
+
+To configure the Serverless chart, override the default values of its `values.yaml` file. This document describes parameters that you can configure.
+
+>**TIP:** To learn more about how to use overrides in Kyma, see the following documents:
+>* [Helm overrides for Kyma installation](/root/kyma/#configuration-helm-overrides-for-kyma-installation)
+>* [Top-level charts overrides](/root/kyma/#configuration-helm-overrides-for-kyma-installation-top-level-charts-overrides)
+
+## Configurable parameters
+
+This table lists the configurable parameters, their descriptions, and default values for both cluster and local installations.
+
+>**CAUTION:** Currently, there is no automated validation for CPU and memory requests and limits. If you update their default values, make sure that the values for CPU and memory requests are not lower than the default ones, and that values for requests are lower than or equal to those for the limits.
```

Comment:
I'm not sure if you mention building resources or runtime resources? So I'm also not sure what does it mean? `If you update their default values, make sure that the values for CPU and memory requests are not lower than the default ones`

Reply:
I will specify that this concern building resources.